#160520896 ft-get-all-orders

Johnpaul-Borree opened the pull request:
#### What does this PR do?
- Enable users to view orders history
#### Description of Task to be completed?
- create controller actions to get all orders history
- create a route for this endpoint
- display messages on json string
#### How should this be manually tested?
- Test API endpoint on postman using Get /api/v1/orders
- but before you do so, npm run start
#### What are the relevant pivotal tracker stories?
#160520896

KaiserPhemi reviewed (commented):
#### Nitpick
Please fix the title of the PR (and all PRs going forward) to follow best practices/convention

KaiserPhemi commented on src/controller/ordersControllers.js:
```diff
@@ -1,37 +1,40 @@
-    return Joi.validate(order, schema);
+	return Joi.validate(order, schema);
     
 }
```
#### Nitpick
Add a new line

Johnpaul-Borree replied:
Thanks, done

KaiserPhemi commented on src/controller/ordersControllers.js:
```diff
@@ -1,37 +1,40 @@
+	res.json({justAdded: order, message: 'order Created'});
 }
 
 function validateOrders(order) {
```
#### Critical
Same feedback on #11

Johnpaul-Borree replied:
Thanks so much, it has been taken care of

KaiserPhemi reviewed (changes requested):
Attend to feedback

Johnpaul-Borree committed:
fix object spaces and new line

Johnpaul-Borree committed:
**Implement feedback**

- Refactor code using Airbnb style guide
- Add newline at end of the code block
- Remove all function keywords
- add spaces in object curly braces

Johnpaul-Borree committed:
Remove all function keywords

Johnpaul-Borree committed:
[#160520896] implement feedback

- Modify .eslintrc.json to follow airbnb style guide
- Refactor code based on airbnb style
- add newline
- add spaces to objects with curly braces.

Johnpaul-Borree committed:
[#160520896] resolve conflict

Johnpaul-Borree committed:
Merge branch 'develop' into ft-get-all-orders-#160520896
